broker/broker.go

```diff
@@ -289,7 +292,7 @@ func main() {
 		metricsFile = os.Stdout
 	}
 
-	metricsLogger := log.New(metricsFile, "", log.LstdFlags|log.LUTC)
+	metricsLogger := log.New(metricsFile, "", 0)
```

Comment:
Why are we getting rid of the flags here? I'm not saying we shouldn't; I'm just curious.

Reply:
The flags for loggers are mostly about formatting timestamps and prefixes for log messages. We don't want any prefixes for the log file, only the information in the specification.